docker/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.8-slim-buster
```

Comment:
should we go with alpine instead? as it tends to be faster and take less memory.